<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>439</width>
    <height>464</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TCPclient</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" rowspan="7" colspan="4">
     <widget class="QTextBrowser" name="tb_result">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QLabel" name="lb_request">
      <property name="text">
       <string>Запрос</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="7" column="4">
     <widget class="QLabel" name="lb_ip">
      <property name="text">
       <string>IP сервера</string>
      </property>
     </widget>
    </item>
    <item row="5" column="4">
     <widget class="QPushButton" name="pb_clear">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Очистить</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QSpinBox" name="spB_ip3">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximum">
       <number>255</number>
      </property>
     </widget>
    </item>
    <item row="4" column="4">
     <widget class="QLineEdit" name="le_data">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="lb_connectStatus">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="6" column="4">
     <widget class="QPushButton" name="pb_request">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Отправить</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0" colspan="4">
     <widget class="QSpinBox" name="spB_port">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>65535</number>
      </property>
      <property name="value">
       <number>12345</number>
      </property>
     </widget>
    </item>
    <item row="7" column="2">
     <widget class="QSpinBox" name="spB_ip2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximum">
       <number>255</number>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QSpinBox" name="spB_ip4">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="value">
       <number>127</number>
      </property>
     </widget>
    </item>
    <item row="9" column="4">
     <widget class="QPushButton" name="pb_connect">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Подключиться</string>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="8" column="4">
     <widget class="QLabel" name="lb_port">
      <property name="text">
       <string>Порт</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QComboBox" name="cb_request">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <item>
       <property name="text">
        <string>Получить время</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Получить свободное место</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Получить статистику</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Отправить данные</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Очистить память сервера</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="10" column="3">
     <widget class="QFrame" name="sb_frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QFrame" name="sb_fr_status">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="sb_label_status">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="sb_spacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>13</width>
           <height>17</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QFrame" name="sb_fr_hint">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="sb_label_hint">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="7" column="3">
     <widget class="QSpinBox" name="spB_ip1">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="value">
       <number>1</number>
      </property>
     </widget>
    </item>
    <item row="3" column="4">
     <widget class="QLabel" name="lb_image">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>439</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Меню</string>
    </property>
    <addaction name="action_connect"/>
    <addaction name="separator"/>
    <addaction name="action_about"/>
    <addaction name="separator"/>
    <addaction name="action_exit"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <action name="action_connect">
   <property name="text">
    <string>Подключиться</string>
   </property>
  </action>
  <action name="action_about">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
  <action name="action_exit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>tb_result</tabstop>
  <tabstop>spB_ip4</tabstop>
  <tabstop>spB_ip3</tabstop>
  <tabstop>spB_ip2</tabstop>
  <tabstop>spB_ip1</tabstop>
  <tabstop>spB_port</tabstop>
  <tabstop>cb_request</tabstop>
  <tabstop>le_data</tabstop>
  <tabstop>pb_clear</tabstop>
  <tabstop>pb_request</tabstop>
  <tabstop>pb_connect</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pb_clear</sender>
   <signal>clicked()</signal>
   <receiver>tb_result</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>323</x>
     <y>360</y>
    </hint>
    <hint type="destinationlabel">
     <x>242</x>
     <y>348</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
